src/e2e/nuxt.ts

```diff
@@ -14,6 +14,10 @@ const isNuxtApp = (dir: string) => {
     || existsSync(resolve(dir, 'nuxt.config.mjs'))
     || existsSync(resolve(dir, 'nuxt.config.cjs'))
     || existsSync(resolve(dir, 'nuxt.config.ts'))
+    || existsSync(resolve(dir, '.config', 'nuxt.js'))
+    || existsSync(resolve(dir, '.config', 'nuxt.mjs'))
```

Comment:
thank you!

maybe ultimately we might try to get a `resolveConfigPath` into `c12` so we have one location for configuration resolution (cc: @pi0)